__init__.py

```diff
@@ -502,12 +502,13 @@ def handle_mycroft_ready(self, message):
 
     # voice events
     def converse(self, message):
-        if self.state != SetupState.INACTIVE or \
-                self.state != SetupState.FIRST_BOOT:
-            # capture all utterances until paired
-            # prompts from this skill are handled with get_response
-            return True
-        return False
+        if not is_gui_running() or not can_use_touch_mouse():
```

Comment:
should instead check if pairing mode is GUI or not, regardless of mouse etc we dont want this in GUI pairing mode

Reply:
uses pairing mode check now